### DISCUSSION TITLE
Ошибка при роботе дополнения pip discord

### DISCUSSION THREAD

**Discussion by Andrey7894** [Rapptz/discord.py#7353] [Q&A] [answered]
line 94, in wrapped
    raise CommandInvokeError(exc) from exc
discord.ext.commands.errors.CommandInvokeError: Command raised an exception: AttributeError: 'Message' object has no attribute 'guil'

**Comment by ghost** [comment, marked as answer]
It is true. There is no attribute "guil", From this exception i can say that maybe you made mistake, when tried to write "guild".

**Reply by Andrey7894** [reply]
thanks helped
